responsibleai/vision/jobs/prepare.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from azure.ai.ml.entities import AmlCompute, PipelineJob
+from azure.ai.ml import dsl, load_component, MLClient
+from azure.identity import DefaultAzureCredential
+from azureml.core import Workspace
```

Comment:
I think azureml.core does not exist in the test environment, you could either find dpv2 equivalent or update the requirments.txt (can be done after merging this)

https://msdata.visualstudio.com/DefaultCollection/Vienna/_git/azureml-asset?path=/registry/requirements.txt

Reply:
I added some try/except logic, this should not be an issue in the builds